.changeset/warm-terms-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-stuff-core": major
```

Comment:
Is this really a major release? It should be backward-compatible, no?

Reply:
Yeah, probably not.